pkg/models/testcase.go

```diff
@@ -36,7 +36,7 @@ type TestCase struct {
 	GrpcResp GrpcResp            `json:"grpcResp"`
 	GrpcReq  GrpcReq             `json:"grpcReq"`
 	Anchors  map[string][]string `json:"anchors"`
-	Noise    []string            `json:"noise"`
+	Noise    map[string][]string            `json:"noise"`
```

Comment:
correct the indentation in all the files.

Reply:
done!

Reply:
I can see many indentation error still. You can self-review the files once .